### PR TITLE
CONTRAST-20368 Visit your org link should take the use tot Teamserver…

### PR DESCRIPTION
**Pull request by serge17**
"Visit your org" link now takes the user to Teamserver in a new tab.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,12 +48,13 @@ <h2 class="heading">Activity Feed</h2>
       </div>
 
       <div id="get-started">
-        <img src="img/getStarted.png" id="get-started-img"/>
+        <img src="img/getStarted.png" id="get-started-img" />
         <div id="get-started-description" align="center">
-          <a href="https://app.contrastsecurity.com/Contrast">Visit your organization on Teamserver</a> to onboard some spplications.
+          <a id="visitOrg">Visit your organization on Teamserver</a> to onboard some spplications.
```
Comment (gmoore) [inline index.html]: html ids, at least for the moment, are dashed not camelcased

Comment (serge17) [reply index.html]: ID changed

**Comment by gmoore** [comment]
@ryandens @serge17 That's a good point. What should we do if there's a poorly formatted Teamserver URL? I'd hate to have someone fat-finger the URL, get no activities, then get a 404 when clicking the Teamsevrer link. Poor experience.

**Comment by serge17** [comment]
@ryandens @gmoore Teamserver url is now formatted before saving, so this should work now: "app.contrastsecurity.com".

**Comment by gmoore** [comment]
teamserver_link

**Comment by serge17** [comment]
@gmoore Fixed

**Comment by gmoore** [comment]
teamserver-link-404

**Comment by serge17** [comment]
@gmoore I could not reproduce the problem shown on the gif. It seems that the only way it could appear is if a wrong teamserver url was saved to settings. It looks like it opens "chrome-extension://" url.

**Pull request merged**